### PR TITLE
Update free-courses-hi.md

### PR DESCRIPTION
**Pull request by ivaibhavraheja**
added tutorial in for git & github and mogodb

## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by sohansai** [comment]
Insert your links in alphabetical order, following the guidelines in [CONTRIBUTING.md](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -423,6 +424,7 @@
 * [MongoDB Tutorial for Beginners (Hindi)](https://www.youtube.com/playlist?list=PLgPJX9sVy92xUxpTFgAOSBHdBwIdxav39) - CS Geeks
 * [MongoDB tutorial in hindi](https://www.youtube.com/playlist?list=PLolI8AY2AS9aaE4Vx0adwfwUh3XiuVewX) - Code Improve
 * [MongoDB Tutorial in Hindi 2022](https://www.youtube.com/playlist?list=PLQDioScEMUhkcqbgJ4ap2k4zg3sT_-Bbc) - Programming Experience
+* [MongoDB full course](https://youtube.com/playlist?list=PLA3GkZPtsafZydhN4nP0h7hw7PQuLsBv1&si=fj9OK4DshAlDpJd_) - Engineering Digest
```
Comment (eshellman) [inline courses/free-courses-hi.md:427]:
```suggestion
* [MongoDB full course](https://www.youtube.com/playlist?list=PLA3GkZPtsafZydhN4nP0h7hw7PQuLsBv1&si=fj9OK4DshAlDpJd_) - Engineering Digest
```

File: courses/free-courses-hi.md [hunk 2]
```diff
@@ -255,6 +255,7 @@
 * [Github \| All about Git and GitHub](https://www.youtube.com/watch?v=77b2lVHHZqI) - Anuj Bhaiya
 * [GitHub Tutorial in Hindi](https://youtube.com/playlist?list=PLVdoaEL574VBxxcGQmTjxS-JoP5rKV8Wi) - Be A Programmar
 * [GitHub with Visual Studio (Hindi)](https://www.youtube.com/playlist?list=PLbGui_ZYuhigWA1mNWzwErSBIZvgOJbNc) - Rajesh Kumar, Geeky Shows
+* [Complete Git and Github tutorial](https://youtu.be/Ez8F0nW6S-w?si=y4V7MYwZ7bnjduCz) - Apna College
```
Comment (eshellman) [inline courses/free-courses-hi.md:258]:
```suggestion
* [Complete Git and Github Tutorial for Beginners](https://www.youtube.com/watch?v=Ez8F0nW6S-w) - Apna College
```
title from resourse

File: courses/free-courses-hi.md [hunk 3]
```diff
@@ -423,6 +424,7 @@
 * [MongoDB Tutorial for Beginners (Hindi)](https://www.youtube.com/playlist?list=PLgPJX9sVy92xUxpTFgAOSBHdBwIdxav39) - CS Geeks
 * [MongoDB tutorial in hindi](https://www.youtube.com/playlist?list=PLolI8AY2AS9aaE4Vx0adwfwUh3XiuVewX) - Code Improve
 * [MongoDB Tutorial in Hindi 2022](https://www.youtube.com/playlist?list=PLQDioScEMUhkcqbgJ4ap2k4zg3sT_-Bbc) - Programming Experience
+* [MongoDB full course](https://www.youtube.com/playlist?list=PLA3GkZPtsafZydhN4nP0h7hw7PQuLsBv1&si=fj9OK4DshAlDpJd_) - Engineering Digest
```
Comment (eshellman) [inline courses/free-courses-hi.md:427]:
```suggestion
* [MongoDB Playlist in Hindi](https://www.youtube.com/playlist?list=PLA3GkZPtsafZydhN4nP0h7hw7PQuLsBv1&si=fj9OK4DshAlDpJd_) - Engineering Digest
```

**Pull request merged**